packages/models-library/src/models_library/projects.py

```diff
@@ -141,7 +159,15 @@ class Node(BaseModel):
         example=["nodeUUid1", "nodeUuid2"],
     )
 
-    position: Position = Field(...)
+    position: Optional[Position] = Field(...)
+
+    class Config:
+        extra = Extra.forbid
+
+
+class StudyUI(BaseModel):
+    workbench: Optional[Dict[NodeID, WorkbenchUI]] = Field(...)

```

Comment:
This again is kind of contradicting:  ``Optional`` (not required) means exactly the opposite to ``Field(...)`` (i.e. required)

There are many options, just check about [required fields](https://pydantic-docs.helpmanual.io/usage/models/#required-fields) and decide which one you want

Reply:
The Position in the workbench field will be stored from now on in ui.workbench. That's why I just marked it as deprecated. The frontend will take care of moving the Position field to the right place.